cayleypy/graphs_lib_test.py

```diff
@@ -21,6 +21,80 @@ def test_pancake():
     ))
 
 
+def test_cubic_pancake():
+
+    graph = prepare_graph("cubic_pancake", n=15, S=1)
+    assert graph.n_generators == 3
+    assert graph.generator_names == ["R15", "R14", "R2"]
+    assert torch.equal(graph.generators, torch.tensor(
+        [
+            [14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0],
+            [13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0, 14],
+            [1, 0, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14]
+        ]
+    ))
+    graph = prepare_graph("cubic_pancake", n=15, S=2)
```

Comment:
Please separate blocks with one empty line, so it is clear where new test case starts.